hw_oop/ex1.py

```diff
@@ -0,0 +1,31 @@
+class Shape:
+
+    def __init__(self, w, h):
+        self.width = w
+        self.height = h
+
+
+class Triangle(Shape):
+    def __init__(self, w, h):
```

Comment:
This is the complete copy of init in base class, should be removed if no changes made.